<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="renderArea" name="render_area">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="coef_a">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="coef_b">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="coef_c">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>renderArea</class>
   <extends>QFrame</extends>
   <header>renderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
